libraries/lib-wave-track/CachingPlayableSequence.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "AudioIOSequences.h"
+#include "AudioSegmentSampleView.h"
+
+#include <memory>
+
+class WaveTrack;
+
+class WAVE_TRACK_API CachingPlayableSequence : public PlayableSequence
+{
```

Comment:
Some Doxygen commentary please about the purpose of the class.

Reply:
Why on this particular class? Class comments are the exception in this repo, if there's a rule of thumb I'd like to know it (besides that's it's a cool class).

Reply:
Comment it at least because the class is meant for use outside the library that defines it.  I don’t think you must comment every type defined in anonymous namespaces.

If many classes don’t have DOxygen, that’s not an excuse for improving practice now.

Reply:
> If many classes don’t have DOxygen, that’s not an excuse for improving practice now.

That wasn't my intention, but I don't want to feel I have to comment every single class where not necessary.

The visibility-beyond-hosting-library sounds like a good rule of thumb to me.